From 95f4651a793030ec7ca60c636f4085f7d4dbffff Mon Sep 17 00:00:00 2001
From: Tim Foster <tim.foster@joyent.com>
Date: Fri, 8 Mar 2019 15:44:11 +0000
Subject: [PATCH] MANTA-4161 more manta repositories should not use git+ssh://
 URIs in package.json files Reviewed by: Kelly McLaughlin
 <kelly.mclaughlin@joyent.com> Approved by: Kelly McLaughlin
 <kelly.mclaughlin@joyent.com>

---
 package.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package.json b/package.json
index 52b98ff..d450502 100644
--- a/package.json
+++ b/package.json
@@ -14,7 +14,7 @@
         "bunyan": "0.21.4",
         "cmdutil": "1.1.0",
         "dashdash": "1.3.2",
-        "libmanta": "git+ssh://git@github.com:joyent/node-libmanta.git#master",
+        "libmanta": "https://github.com/joyent/node-libmanta.git#master",
         "moray": "3.3.0",
         "node-uuid": "1.4.0",
         "once": "1.2.0",
-- 
2.21.0

